_abouts/masthead.md

```diff
@@ -5,7 +5,7 @@ permalink: /about/masthead/
 banner: true
 ---
 
-<div class="intro">
+<div class="intro masthead">
   <div class="row">
     <ul class="col s6 m6 l4 collection with-header">
```

Comment:
ohey @whatcoloris i think what would work better than this new `!important` css class would be to not have these collections sized `s6` but rather `s12` so a single column takes up the full screen width on small media sizes.

also, the sizes on this have gotten a little weird. and there's now a bunch of dead space on this page. see screenshot 
![Screen Shot 2019-04-07 at 2 18 05 PM](https://user-images.githubusercontent.com/560568/55687900-34324700-5940-11e9-95c1-c73992528841.png)

i think we should change `col s6 m6 l4 collection with-header` to `col s12 m6 collection with-header` for these collections. and change the `col s12 m12 l9` to `col s12` for the wider collections (Past Editors & so-on) below.

Reply:
make whatever changes you see fit mah freund!

Reply:
okay. but this is your PR 😉

Reply:
oh riiiiiiiiiight code of schmonducks